### PR TITLE
Migrate infra to non-worker nodes

### PR DESCRIPTION
**Pull request by RobertKrawitz**
Not yet tested.

### CODE REVIEW

File: benchmark_runner/common/ocp_resources/migrate_infra.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    def create_odf(self):
+        """
+        This method create odf
+        :return:
+        """
+        for resource in self.__resource_list:
+            logger.info(f'run {resource}')
+            if resource == '01_cluster-monitoring-configmap-template.yaml':
+                nodes = self.__oc.get_master_nodes().split()
+                if len(nodes) < 1:
```
Comment (portante) [inline benchmark_runner/common/ocp_resources/migrate_infra.py:30]: What is the purpose of this test?

Comment (RobertKrawitz) [reply benchmark_runner/common/ocp_resources/migrate_infra.py]: If we don't have any infra nodes, we don't want to try to bind the infra components to infra nodes.

File: benchmark_runner/common/ocp_resources/migrate_infra.py [hunk 2]
```diff
@@ -27,9 +26,8 @@ def create_odf(self):
             logger.info(f'run {resource}')
             if resource == '01_cluster-monitoring-configmap-template.yaml':
                 nodes = self.__oc.get_master_nodes().split()
-                if len(nodes) < 1:
-                    pass
-                for node in nodes:
-                    self.run(f'oc label {node} node-role.kubernetes.io/infra=')
-                self.__oc._create_async(yaml=os.path.join(self.__path, resource))
+                if len(nodes) >= 1:
```
Comment (portante) [inline benchmark_runner/common/ocp_resources/migrate_infra.py:29]: FWIW, since `.split()` returns a `list`, `if nodes:` is sufficient.

**Pull request merged**